### PR TITLE
Adds in the Melee weapons

### PR DESCRIPTION
**Pull request by ProfessorPopoff**
Uhhh. **Yo.**

Submitting this as a draft for now. I've got it along decently, I'm mainly trying to iron out exactly how I can get all the wielded weapons to function, as well as some of the special weapons. Roughly half done, will update this as I go along. For now, witness peak badcode and the awesome sprites provided by Infra, along with a bit of sprinkled in certified codersprite to compensate for the lack of toggled variants. 
image
image

### CODE REVIEW

File: mojave/items/melee/axe.dm [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//This file contains all axes and their subtypes, like Sledgehammers.
+
+/obj/item/fireaxe/ms13
+	name = "fire Axe"
```
Comment (Hekzder) [inline mojave/items/melee/axe.dm:4]:
```suggestion
	name = "fireaxe"
```

File: mojave/items/melee/knife.dm [hunk 2]
```diff
@@ -11,29 +15,90 @@
 
 /obj/item/kitchen/knife/ms13/combat
 	name = "combat knife"
-	desc = "A well made, serrated combat knife. Very effective at wounding."
-	force = 30
+	desc = "A well made, serrated combat knife. Very effective at wounding. It's got a light frame, with a rifle lug to potentiall attach to a matching rifle."
```
Comment (Hekzder) [inline mojave/items/melee/knife.dm:18]:
```suggestion
	desc = "A well made, serrated combat knife. Very effective at wounding. It's got a light frame, with a rifle lug to potentially attach to a matching rifle."
```

File: mojave/items/melee/knife.dm [hunk 3]
```diff
@@ -11,29 +15,90 @@
...
 	name = "bowie knife"
 	desc = "A heavy duty bowie knife. Not as good at wounding as it's serrated counterparts, but can kill just as well."
+	icon_state = "knife_bowie"
 	force = 35
 	armour_penetration = 5
 	wound_bonus = -4
 	throwforce = 35
 
+/obj/item/kitchen/knife/ms13/hunting
+	name = "hunting"
```
Comment (Hekzder) [inline mojave/items/melee/knife.dm:43]:
```suggestion
	name = "hunting knife"
```

**Review by Hekzder — CHANGES_REQUESTED** [review]
Just some minor typo level stuff.

**Comment by ProfessorPopoff** [comment]
Well well well. The time approaches. I think I've just about got everything done here. I'm opening this for review, may god have mercy. I added everything to the best of my ability and also learned many things over the last half month. God be praised it's over, though. Overall, this has added a TON of melee weapons of varying types. Hekzder will probably want to comb over this one, as I have even less of an idea of what the values do than I do with the armours. 

image
image
image

As you can see, the brick and the flare are T4. Direct further questions to Koshenko. Good day.

**Comment by Mannybrado** [comment]
hm... but did you make a google spreadsheet calculating all of the damage, dps, attack delay, armor piercing, and wound damage all of them do to multiple types of armor of every tier?

**Comment by Mannybrado** [comment]
> Well well well. The time approaches. I think I've just about got everything done here. I'm opening this for review, may god have mercy. I added everything to the best of my ability and also learned many things over the last half month. God be praised it's over, though. Overall, this has added a TON of melee weapons of varying types. Hekzder will probably want to comb over this one, as I have even less of an idea of what the values do than I do with the armours.
> 
> image
> image
> image
> 
> As you can see, the brick and the flare are T4. Direct further questions to Koshenko. Good day.

btw you included the normal hammer in the twohand category :\\\\\\\\\\\\\\\\\\\\\

**Comment by ProfessorPopoff** [comment]
> btw you included the normal hammer in the twohand category :\\\\\\\\\\\

Meh. It happens.

**Review by Hekzder — APPROVED** [review]
Balance TBD but there isn't much of a reason not to merge this so everything is at least in the files and more organized.

**Pull request merged**